helpers/httpApi.js

```diff
@@ -192,6 +194,26 @@ function respond (res, err, response) {
 }
 
 /**
+ * Adds code status to responses for every failed request.
+ * Default error code is 500.
+ * Success code is 200.
+ * Success code for empty data is 204.
+ * @param {Object} res
+ * @param {ApiError} err
+ * @param {Object} response
+ */
+function respondWithCode (res, err, response) {
+	if (err) {
+		res.code(err.code || apiCodes.ERROR_DEFAULT).json(err.toJson());
+	} else {
+		var isResponseEmpty = function (response) {
+			var firstValue = _(response).values().first();
+			return _.isArray(firstValue) && _.isEmpty(firstValue);
+		};
+		return res.code(isResponseEmpty(response) ? apiCodes.EMPTY_RESOURCES_SUCCESS : apiCodes.SUCCESS).json(response);
```

Comment:
In case of non-empty resource, we only return status code `api.Codes.SUCCESS` and no other status code?